client/src/components/ConfigTemplates/InstanceDropdown.vue

```diff
@@ -0,0 +1,58 @@
+<script setup lang="ts">
+import "./icons";
```

Comment:
Please avoid side-effect imports.
To reduce duplicated code, use a composable instead.

Reply:
This is a pattern we use other places in the code. I will just unwind the imports and place them in the target files.